sky/clouds/service_catalog/kubernetes_catalog.py

```diff
@@ -0,0 +1,31 @@
+"""Kubernetes Catalog.
+
+Kubernetes does not require a catalog of instances, but we need an image catalog
+mapping SkyPilot image tags to corresponding container image tags.
```

Comment:
```suggestion
mapping SkyPilot image tags to corresponding container image ids.
```

Reply:
Hmm, they are still container image tags (e.g., `ubuntu:latest`), since container image ids are hex values and are typically not used to specify images to use.